src/icos_config/gladius.js

```diff
@@ -0,0 +1,87 @@
+import { toPromise } from '../utils';
+import { convertWeb3Value } from '../utils/web3';
+
+export default {
+  // crowdSaleTokenContract: '0x57BFfFD48366F78e787e167419C8c05CDb849EdE',
+  // tokenContract: '0x4632d1c31c5D9E28E84eAE0173B3aFc9acA81aC8',
```

Comment:
Why are you leaving these commented code?

Reply:
Ok I will clarify why this is left